multicast/exceptions.py

```
@@ -629,7 +728,7 @@
 	"""
 
 	@functools.wraps(func)
-	def wrapper(*args, **kwargs):
+	def wrapper(*args, **kwargs) -> callable:
```

Ruff (0.8.2)

Missing type annotation for `*args`

(ANN002)